tic-tac-toe-the3rd/src/components/Game.js

```diff
@@ -0,0 +1,96 @@
+import React, { useContext } from 'react';
+import Cells from './Cell';
+import BoardContext from '../context/BoardContext';
+
+
+function Board(props) {
+  const { board, setBoard, turn, setTurn, history, setHistory } = useContext(BoardContext);
```

Comment:
Using a context here is a cool choice! I think I'd be inclined to put even more logic into the context and let the board know even less. For example, I think your `declareWinner` and `clickHandler`, and `travel` functions could all go onto the context. It's possible that they could then replace the `setBoard`, `setTurn`, and `setHistory` functions. Basically, you can let the context do more of the heavy lifting. Let me know if this seems to make sense!